src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -183,6 +183,13 @@ def safe_to_destroy(self, svc_id):
 
 @ApiController('/osd/flags', Scope.OSD)
 class OsdFlagsController(RESTController):
+    def __init__(self):
+        self.unsupported_set_unset_flags = [
```

Comment:
I like this: moving knowledge about Ceph closed to the core, and making the front-end more agnostic.